Ux changes

anaganisk opened the pull request:
Hey I made some UX changes. 
Totally upto you if you want to merge.
A quick summary
- Output is streamed in html rather than JSON dump.
- Split the main container into 2 column flex. 
- made the model name env variable for switching models (make sure u add the environment variable. PUBLIC_GOOGLE_MODEL="")
- I ran the formatter included in your package.json, so a few files have the formatting applied.

![preview changes](https://github.com/user-attachments/assets/be108e51-7327-4720-8a39-439a0232d0d4)

mikeesto reviewed (commented):
Thanks for the PR @anaganisk . I like the improved JSON parsing and reset button. If you'd like to open a PR with just those changes, I'd be happy to merge it. Otherwise, the project is available under MIT if you'd like to maintain a fork with your preferred UI etc : )

anaganisk commented:
Sure, I will do.
Its simple enough to be a single project, don't think a fork is necessary for now.

Commit:
Ux changes

anaganisk commented:
@mikeesto I updated the PR only with the UX changes without other formatting changes, I also noticed I forgot adding the footer earlier. I added it to the main card.

<img width="1470" alt="image" src="https://github.com/user-attachments/assets/4620302a-da52-41bc-9473-2e999b8f10db" />
